src/Configuration/test/CloudFoundry.ServiceBinding.Test/PostProcessorsTest.cs

```diff
@@ -259,4 +259,35 @@ public void Processes_Eureka_configuration()
         configurationData[$"{keyPrefix}:AccessTokenUri"].Should().Be("test-access-token-uri");
         configurationData[$"{keyPrefix}:Enabled"].Should().Be("true");
     }
+
+    [Fact]
+    public void Processes_Identity_configuration()
+    {
+        var postProcessor = new IdentityCloudFoundryPostProcessor(NullLogger<IdentityCloudFoundryPostProcessor>.Instance);
+
+        var secrets = new[]
+        {
+            Tuple.Create("credentials:auth_domain", "test-domain"),
+            Tuple.Create("credentials:client_id", "test-id"),
+            Tuple.Create("credentials:client_secret", "test-secret"),
+
+            // these are included in bindings, but not currently mapped:
```

Comment:
we could probably do some validation to help if bindings aren't created right, but I don't have a significant use case. I'll delete since we have raw JSON examples elsewhere